external: fix external cluster docs (#15004)
rook/rook: Documentation/CRDs/Cluster/external-cluster/provider-export.md

## 1. Create all users and keys

> Run the python script [create-external-cluster-resources.py](https://github.com/rook/rook/blob/master/deploy/examples/external/create-external-cluster-resources.py) in external Ceph cluster cephadm shell, for creating all users and keys.

Reviewer: ```suggestion
Run the python script [create-external-cluster-resources.py](https://github.com/rook/rook/blob/master/deploy/examples/external/create-external-cluster-resources.py) in the provider Ceph cluster cephadm shell, to have access to create the necessary users and keys.
```
